### ISSUE TITLE
Aborted when on_state_create ~= nil

### ISSUE THREAD

**Issue by betabrain** [LuaLanes/lanes#139]
I cannot get `on_state_create` to work with a Lua function. Here is my test code:

```lua
local lanes = require("lanes").configure({
    on_state_create = function () end,
})

local lanegen = lanes.gen(function () return true end)
local lane_h = lanegen()
print(lane_h:join())
```

Result:

```
$ lua example-osc.lua
Aborted
```

Versions:

```
$ luarocks list

Rocks installed for Lua 5.3
---------------------------

lanes
   3.10.1-1 (installed) - /usr/local/lib/luarocks/rocks-5.3

$ lua -v
Lua 5.3.5  Copyright (C) 1994-2018 Lua.org, PUC-Rio
```

**Comment by benoit-germain** [comment]
It looks like there is a bug in Lanes when the lane doesn't initialize any library. The error goes away when changing your code to
```lua
local lanegen = lanes.gen( "*", function () return true end)
```

**Comment by benoit-germain** [comment]
Fixed by 94c794c8e06a848868509927eefde0741cd8bbe6.
